notebooks_tsqr/NightLog.ipynb

```diff
@@ -0,0 +1,755 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #67.            url_http_status_code = dict()
</pre><p>see comment about <code>list()</code> versus [] for dict() and {} as well please </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_logging_and_reporting/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>